scripts/custom/core.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# core.py core icon theme custom install script
+#
+# Copyright (C) 2015  Stefan Ric (cybre)
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA
+import sys
+import os
+from os import symlink
```

Comment:
It's just style, instinct :). It doesn't affect anything. I like having some parts of the library called like `module.fuction()` if I think I want to know at all times which module owns the function and some with just `function()` if I don't care what module the function comes from. Just like I said, I do it instinctively .